cephci/utils/configs.py

```diff
@@ -0,0 +1,99 @@
+import yaml
+
+from cli.exceptions import ConfigError
+
+DEFAULT_CONFIG_PATH = "~/.cephci.yaml"
+CONFIG = None
+
+
+def get_configs(config=DEFAULT_CONFIG_PATH):
+    global CONFIG
+    if CONFIG:
+        return CONFIG
+
+    with open(config, "r") as _stream:
+        try:
+            CONFIG = yaml.safe_load(_stream)
+        except yaml.YAMLError:
+            raise ConfigError(f"Invalid configuration file '{config}'")
+
+
+def _get_credentials():
+    if not CONFIG:
+        raise ConfigError("Configuration is not passed")
+
+    try:
+        return CONFIG["credentials"]
+    except KeyError:
+        raise ConfigError("Credentials configurations are missing from config")
+
+
+def get_subscription_credentials(server):
+    _dict = {}
+    try:
+        _subscription = _get_credentials()["subscription"]
+        _server = _subscription[server]
+        _dict["serverurl"] = _server["serverurl"]
+        _dict["baseurl"] = _server["baseurl"]
+        _dict["username"] = _server["username"]
+        _dict["password"] = _server["password"]
+        _dict["timeout"] = int(_server.get("timeout", 60))
+        _dict["retry"] = int(_server.get("retry", 6))
+    except KeyError:
+        raise ConfigError(f"Insufficient config for '{server}' in subscription")
+
+    return _dict
+
+
+def get_registry_credentials(server, build):
```

Comment:
same here, the server variable has ibm or rh as the value, so can it be renamed as well?

Reply:
here server will have the values like `cdn`, `stage`, `cp` etc and build is `rh` & `ibm`